int8 arm_index
bool relative
float32 vel
float32[] terminal
---
int32 result
string strRt